README.md

```diff
@@ -1,2 +1,16 @@
-# 2017-Hestia-Server
+push# 2017-Hestia-Server
 Home automation server.
+
+Tested on python version 3.5.2
+
+Before use first install `pip`
+`pip install flask`
+`pip install flask_restplus`
+
+Run the webservice using `python3 source/application.py`
+
```

Comment:
Just a suggestion: This could be handled by a simple script, which will make setting up a development environment much easier and fast. Have a look at what, for example, the Mycroft team uses:
https://github.com/MycroftAI/mycroft-core/blob/dev/dev_setup.sh